### ISSUE TITLE
"Not in your wallet", "revoked" and other error text isn't read with voiceover on

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1789]
### Description of problem
With VoiceOver or Taslkaback on and when receiving a proof request asking for credentials a person does not have, or revoked credentials, or credentials that not meet requirements, the "not in your wallet", "revoked", and "requirements not met" text does not get read.

### Expected behavior
- The error text when a proof request does not have the necessary credentials/attributes should be read outload when Talkback or VoiceOver is enabled

### Steps to reproduce
- Turn on voiceover
- Receive a proof request asking for credentials a person does not have, or revoked credentials, or requirements aren't met
- tap on card to read the content with voiceover

### Environment
- [X] Occurs on Android
- [X] Occurs on iOS

Build #: 1527

Android Device Model: Google pixel 8

iOS Device Model: iPhone 12

### Workaround
none

### Severity
- [ ] High
- [X] Medium
- [ ] Low

**Comment by knguyenBC** [comment]
estimated at 3.

**Comment by knguyenBC** [comment]
Still an issue. removed stale label
